aptos-move/block-executor/src/txn_last_input_output.rs

```diff
@@ -180,7 +180,7 @@ impl<K: ModulePath, T: TransactionOutput, E: Send + Clone> TxnLastInputOutput<K,
         txn_idx: TxnIndex,
         input: Vec<ReadDescriptor<K>>,
         output: ExecutionStatus<T, Error<E>>,
-    ) {
```

Comment:
nit: maybe make this an anyhow.result <()>, because I think that's better for indicating whether there was a problem or method succeeded.